stl/inc/vector

```diff
@@ -3720,6 +3720,160 @@ _NODISCARD _CONSTEXPR20 _Iter_diff_t<_VbIt> _Count_vbool(_VbIt _First, const _Vb
         });
 }
 
+template <class _VbIt, class _OutIt>
+_CONSTEXPR20 _OutIt _Copy_vbool(_VbIt _First, _VbIt _Last, _OutIt _Dest) {
+    // copy [_First, _Last) to [_Dest, ...)
+    if (_First == _Last) {
+        return _Dest;
+    }
+
+    _Vbase* _VbFirst    = const_cast<_Vbase*>(_First._Myptr);
+    _Vbase* _VbLast     = const_cast<_Vbase*>(_Last._Myptr);

```

Comment:
why do we need to cast the const off these? Presumably we shouldn't be writing through them anyway.

Reply:
I believe that is simply legacy because of the way `_Myptr` is declared

Reply:
It's true that `_Myptr` is declared to be `const` so we must cast it away for `_VbDest`. However, we should indeed preserve it for first/last (even if other code hasn't been so disciplined). I'll make this change.